pages/content/shared/copy-script.html

```diff
@@ -0,0 +1,44 @@
+$view: /layouts/blank.j2
+$sitemap:
+  enabled: false
+---
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1">
+  <meta name="robot" content="noindex,nofollow">
+  <style>
+    {% do doc.styles.addCssFile('/css/components/atoms/icon.css') %}
+    {{ doc.styles.emit() }}
+    html, body { background: none transparent; }
+    body { display: flex; min-height: 100vh; align-items: center; }
+    svg { fill: #005af0; }
+    input { opacity: 0 }
+  </style>
+</head>
+<body>
+  <svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <defs>
+      {{ doc.icons.emit() }}
```

Comment:
Are these all icons or only the copy one?

Reply:
This will only emit icons added somewhere else in the document by `{% do doc.icons.useIcon %}` - so yes only the clipboard icon.